Shared/Samples/Apply hillshade renderer to raster/ApplyHillshadeRendererToRasterView.swift

```diff
@@ -0,0 +1,93 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyHillshadeRendererToRasterView: View {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// The map that will be shown.
+        let map: Map
+        
+        /// The raster layer in the map.
+        private let rasterLayer: RasterLayer
+        
+        /// The raster renderer.
+        var renderer: HillshadeRenderer {
+            didSet {
+                // When the renderer is updated, update the layer accordingly.
+                rasterLayer.renderer = renderer
+            }
+        }
+        
+        init() {
+            // Gets the raster file URL.
+            let rasterFileURL = Bundle.main.url(forResource: "srtm", withExtension: "tiff", subdirectory: "srtm")!
```

Comment:
Once the ODR issue is fixed, this should become

```suggestion
            let rasterFileURL = Bundle.main.url(forResource: "srtm", withExtension: "tiff")!
```